src/overlays/actors/ovl_En_Crow/z_en_crow.h

```diff
@@ -6,9 +6,20 @@
 
 struct EnCrow;
 
+typedef void (*EnCrowActionFunc)(struct EnCrow*, GlobalContext*);
+
 typedef struct EnCrow {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x14C];
+    /* 0x014C */ Vec3f unk_14C[4];
```

Comment:
As per EffectSs_EnFire, these are the fire effect positions. It calls them firePos.